src/develop/pixelpipe_hb.h

```diff
@@ -94,6 +94,7 @@ typedef struct dt_dev_pixelpipe_t
   dt_dev_pixelpipe_cache_t cache;
   // set to TRUE in order to obsolete old cache entries on next pixelpipe run
   gboolean cache_obsolete;
+  uint64_t runs;
```

Comment:
I would add a comment:
// to compute and report cache hit rate (only for display purpose)

Reply:
done